src/index.html

```diff
@@ -8,7 +8,224 @@
+                  <li style="display:inline">icon2</li>
+                  <li style="display:inline">icon3</li>
+                </ul>
+              </div>
+              <div id="AboutMe_description">
+                <p>paragraph1</p>
+                <p>paragraph2</p>
+              </div>
+            </div>
+            <!--Slide Experience-->
+            <div id="Experience_Container">
+              <div id="Experience_title">
+                <h2 style="text-transform:uppercase">experience</h2>
+              </div>
+              <div >
+                <button style="text-transform:uppercase">download my full Resume</button>
+              </div>
+              <div>
+                <h6>Budget Director@Wix.com</h6>
+                <h6>2014 - Current</h6>
+                <p>description_1</p>
+              </div>
+              <div>
+                <h6>Business Administration@BizzNizz.net</h6>
+                <h6>2010 - 2013</h6>
+                <p>description_2</p>
+              </div>
+              <div>
+                <h6>Finance Director@Computers & More LTD.</h6>
+                <h6>2008 - 2012</h6>
+                <p>description_3</p>
+              </div>
+            </div>
+            <!--Slider Education-->
+            <div id="Education_Container">
+              <div id="Education_title">
+                <h2 style="text-transform:uppercase">education</h2>
+              </div>
+              <div >
+                <button style="text-transform:uppercase">download my full Resume</button>
+              </div>
+              <div>
+                <h6>The San Francisco Business School</h6>
+                <h6>2010 - 2012</h6>
+                <p>description_1</p>
+              </div>
+              <div>
+                <h6>NYC School of Marketing and Advertising</h6>
+                <h6>2007 - 2010</h6>
+                <p>description_2</p>
+              </div>
+              <div>
+                <h6>L.A. University of Applied Sciences</h6>
+                <h6>2002 - 2006</h6>
+                <p>description_3</p>
+              </div>
+            </div>
+            <!--Slider Hobbies-->
+            <div id="Hobbies_Container">
+              <div id="Hobbies_title">
+                <h2 style="text-transform:uppercase">Hobbies & intreset</h2>
+              </div>
+              <div >
+                <button style="text-transform:uppercase">download my full Resume</button>
+              </div>
+              <div>
+                <ul>
+                  <li style="display:inline-block">
+                    <img src="cinema.jpg" alt="icon1">
+                    <p>Cinema</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Sport.jpg" alt="icon1">
+                    <p>Sport</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Theatre.jpg" alt="icon1">
```

Comment:
Don't know if you are just copy and paste the `alt` attribute content, but it would be best to have more meaning for descriptions in it. Screen readers will actually read it since they don't understand any image.

Reply:
Thank you very much for your comment. I will improve this assignment.